test/drenv/__main__.py

```diff
@@ -148,7 +160,10 @@ def start_cluster(profile, **options):
     if is_restart:
         wait_for_deployments(profile)
 
-    execute(run_worker, profile["workers"], hooks=["start", "test"])
+    if skiptest:
+        execute(run_worker, profile["workers"], hooks=["start"])
+    else:
+        execute(run_worker, profile["workers"], hooks=["start", "test"])
```

Comment:
We don't need to duplicte the entire call, the only change is the hook argument.